packages/client/src/types.ts

```diff
@@ -86,40 +85,36 @@ export type setTopicSessionDataFn = (
 /**
  * Topic types
  */
-type BaseTopic = {
+export enum TopicType {
+  PERMIT_CHECK, // A permit-check that is either an "OLO flow" or an "IMTR flow" check. IMTR checks are configured in `packages/imtr/src/config`.
```

Comment:
Why not use different types here?

Reply:
because I like that it is decided by if their topic in the config file and an actual IMTR / JSON file exists